### PR TITLE
Add blender unit details

### PR DESCRIPTION
**Pull request by slimbuck**

### CODE REVIEW

**Review by mvaligursky — APPROVED** [review]

File: docs/user-manual/assets/models/units.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ PlayCanvas scenes generally treat 1 unit as 1 meter.
 
 When authoring artwork for PlayCanvas, it is important to ensure you do so to the desired scale. To do this, you should check the working units for your scene in your modelling application of choice. You can choose whatever working units you like, but just ensure you respect them. For example, to make a cube that is 1 meter in dimension, you could set your working units to meters and create a 1x1x1 cube. Alternatively, you could set working units to centimeters and create a 100x100x100 cube. Exporting either scene to FBX and importing it to PlayCanvas will result in a cube which is 1x1x1.
 
+### Blender
+
+To get ensure units are exported correctly from Blender, check that Scene Properties unit system is set to metric and that scale is set to 1.0:
```
Comment (willeastcott) [inline docs/user-manual/assets/models/units.md:12]:
```suggestion
To ensure units are exported correctly from Blender, check that Scene Properties unit system is set to metric and that scale is set to 1.0:
```

File: docs/user-manual/assets/models/units.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ PlayCanvas scenes generally treat 1 unit as 1 meter.
...
+
+![Blender units](/img/user-manual/assets/models/units/blender-units.png)
+
+In addition, when exporting to FBX format check that "Apply Scaling" is set to "FBX All":
```
Comment (willeastcott) [inline docs/user-manual/assets/models/units.md:16]:
```suggestion
In addition, when exporting to FBX format, check that "Apply Scaling" is set to "FBX All":
```

File: static/img/user-manual/assets/models/units/blender-fbx-export.png
Comment (willeastcott) [inline static/img/user-manual/assets/models/units/blender-fbx-export.png:1]: This image looks much lower res compared to the other one. Can you make them consistent?

Comment (slimbuck) [reply static/img/user-manual/assets/models/units/blender-fbx-export.png:1]: done

**Pull request merged**